Bundle/WidgetMapBundle/Resources/config/services.yml

```diff
@@ -5,9 +5,17 @@ services:
     victoire_widget_map.widget_data_warmer:
         class: Victoire\Bundle\WidgetMapBundle\Warmer\WidgetDataWarmer
         arguments:
-            - "@annotation_reader"
-            - "@victoire_view_reference.repository"
-            - ["\Victoire\Bundle\MediaBundle\Entity\Media"]
+            - @annotation_reader
+            - @victoire_view_reference.repository
+            - @victoire_widget.widget_helper
+            - WidgetSliderItem: WidgetSlider
```

Comment:
I'm not sure about that.
This list help to improve WidgetDataWarmer efficiency but don't have to be exhaustive and precise :
- if an association added here doesn't exist anymore in a Widget, the warmer won't care
- if an association is missing here, it will only generate a couple of useless Doctrine queries

I suggest not to add new config on each concerned Widget and to keep this list here. Is it ok for you ?